### DISCUSSION TITLE
命令行如何隧道高级配置

### DISCUSSION THREAD

**Discussion by DeshunMie** [natfrp/wiki#132] [使用问题]
Linux下的frpc_Linux_amd64 TUI可以加载修改后的frpc.ini，那么命令行启动如何加载自定义配置呢？apache启用proxy protocol需要自定义配置proxy protocol，但是命令行启动并没有使用配置，tui又无法服务器启动

**Comment by DeshunMie** [comment]
查看了frp上游文档，问题已经解决了。
通过参数-c可以加载配置

[Unit]
Description=Sakura Frp
After=network.target syslog.target
Wants=nework.target

[Service]
ExecStart=/home/frp/frpc_linux_amd64 -c /home/frp/frpc.ini
 
[Install]
WantedBy=default.target

这样开机服务启动就不会报错了
![1.png](https://user-images.githubusercontent.com/66585798/183592015-5592f77c-a1a7-4993-9d02-e6df182ec3da.png)

**Reply by fakeboboliu** [reply]
你好，
你可以在「高级用户模式」（在用户信息页面开启）下，在「隧道自定义设置」中设置相关选项。
即可在不使用配置文件的情况下启用此设置了。

如果使用 -c 参数启动，代表您自认为是一位足够极客的用户，后续此设置中出现的问题我们不会处理。

**Reply by DeshunMie** [reply]
感谢解答，之前一直没找到设置位置才只好使用-c参数了
